### PR TITLE
Added user details and time of archive in the more details modal

### LINKED ISSUE #4180: Add the name of user and time details of Archived File to the Modal in File Upload Modal
**Describe the bug**
Add the name of user and time details of Archived File to the Modal in File Upload Modal

**To Reproduce**

1. Go to https://care.coronasafe.in/facility/a31e6a24-0146-4043-b295-3a6c2fe7e095/patient/dcc71a18-2b81-4d56-92c3-f4455aea4b0c/consultation/a014089f-ed53-4639-9b42-27ba8647f41f/files
2. Click on Archived Files Go to next Page
3. Click on More Details

**Expected behavior**
The name of user who archived should be visible who archived the file and the time when the file was archived in More Details Modal
a backend PR is in progress for same https://github.com/coronasafe/care/pull/1109
**Screenshots**
image

### PR DESCRIPTION
**Pull request by patelaryan7751**
## Proposed Changes

- Fixes #4180 
Added user details and time of archive in the more details modal
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@patelaryan7751 can you merge this with the develop branch and after merging if cypress fails, please do resolve that

**Pull request merged**
